### ISSUE TITLE
Error try to run monitoring

### ISSUE THREAD

**Issue by harshhemani** [openai/gym#9]
When trying to run following code on Ubuntu MATE 14.04 LTS on intel i7:
```
import gym
env = gym.make("CartPole-v0")
env.monitor.start("/tmp/cartpole-experiment-1")
for i_episode in xrange(20):
    observation = env.reset()
    for t in xrange(100):
        env.render()
        print observation
        action = env.action_space.sample()
        observation, reward, done, info = env.step(action)
        if done:
            print "Episode finished after {} timesteps".format(t+1)
            break

env.monitor.close()
```

I get following error:
```
[2016-04-28 10:59:12,799] Making new env: CartPole-v0
[2016-04-28 10:59:12,800] Creating monitor directory /tmp/cartpole-experiment-1
[2016-04-28 10:59:12,801] Starting new video recorder writing to /tmp/cartpole-experiment-1/openaigym.video.0.8846.video000000.mp4
avconv version 9.18-6:9.18-0ubuntu0.14.04.1, Copyright (c) 2000-2014 the Libav developers
  built on Mar 16 2015 13:19:10 with gcc 4.8 (Ubuntu 4.8.2-19ubuntu1)
Traceback (most recent call last):
  File "temp.py", line 5, in <module>
    observation = env.reset()
  File "/usr/local/lib/python2.7/dist-packages/gym/core.py", line 87, in reset
    self.monitor._after_reset(observation)
  File "/usr/local/lib/python2.7/dist-packages/gym/monitoring/monitor.py", line 237, in _after_reset
    self.video_recorder.capture_frame()
  File "/usr/local/lib/python2.7/dist-packages/gym/monitoring/video_recorder.py", line 111, in capture_frame
    self._encode_image_frame(frame)
  File "/usr/local/lib/python2.7/dist-packages/gym/monitoring/video_recorder.py", line 161, in _encode_image_frame
    self.encoder.capture_frame(frame)
  File "/usr/local/lib/python2.7/dist-packages/gym/monitoring/video_recorder.py", line 284, in capture_frame
    self.proc.stdin.write(frame.tobytes())
AttributeError: 'numpy.ndarray' object has no attribute 'tobytes'
[2016-04-28 10:59:13,196] Finished writing results. You can upload them to the scoreboard via gym.upload('/tmp/cartpole-experiment-1')
```

Clearly the frame returned by `encoder.capture_frame()` is storing returning ndarray in the `frame`. How can I solve this?
PS: Without monitor, code runs perfectly.

**Comment by rodrigob** [comment]
Please check that you are using a recent version of numpy.

Seems that tobytes was introduced in 1.9 back in 2014.
https://github.com/numpy/numpy/issues/6163

**Comment by harshhemani** [comment]
Updates numpy, and it worked.
Also, tried fix#9 with old numpy, and it still worked.
Thanks.
